src/styles/blocks/card.scss

```diff
@@ -0,0 +1,124 @@
+@import "../utils/variables";
+
+.card {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 944px;
+  margin: 0 auto;
+  padding: 50px 0;
+
+  &__list {
+    margin-bottom: 46px;
+
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    align-content: center;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__item {
+    max-width: 200px;
+    display: flex;
+    flex-direction: column;
+    border-radius: 5px;
+    background-color: #fff;
+    border: 1px solid #f3f3f3;
```

Comment:
Instead of using a hardcoded color code, consider using a variable for the border color.